src/android/CameraLauncher.java

```diff
@@ -386,12 +387,10 @@ private void processResultFromCamera(int destType, Intent intent) throws IOExcep
             FileHelper.stripFileProtocol(this.croppedUri.toString()) :
             FileHelper.stripFileProtocol(this.imageUri.toString());
 
-        if (this.encodingType == JPEG) {
-            try {
-                //We don't support PNG, so let's not pretend we do
-                exif.createInFile(sourcePath);
-                exif.readExifData();
-                rotate = exif.getOrientation();
+            //We don't support PNG, so let's not pretend we do
```

Comment:
I don't think this change should be in scope of this PR (I know there is a person working on .PNG support so this will just bring merge conflict)